spec/neo4j/http/relationship_client_spec.rb

```diff
@@ -65,6 +65,46 @@
     end
 
     it "updates attributes on an existing relationship" do
+      create_node(from)
+      create_node(to)
+
+      relationship = Neo4j::Http::Relationship.new(label: "KNOWS", uuid: "RelationshipUuid", age: 21)
+      create_relationship(from, relationship, to)
+
+      updated_relationship = Neo4j::Http::Relationship.new(label: "KNOWS", uuid: "RelationshipUuid", age: 33)
+      result = create_relationship(from, updated_relationship, to)
+
+      expect(result["relationship"].keys).to eq(%w[uuid age _neo4j_meta_data])
+      expect(result["relationship"]["uuid"]).to eq("RelationshipUuid")
+      expect(result["relationship"]["age"]).to eq(33)
+
+      rel = Neo4j::Http::Relationship.new(label: "KNOWS")
+      relationships = client.find_relationships(relationship: rel, from: from, to: to)
+      expect(relationships.count).to eq(1)
+    end
+
+    it "allows relationships with same labels between same nodes if primary key is set and different" do
+      create_node(from)
+      create_node(to)
+
+      relationship_friend = Neo4j::Http::Relationship.new(label: "KNOWS", primary_key_name: "uuid", uuid: "FriendUuid", how: "friend")
+      edge_a = create_relationship(from, relationship_friend, to)
+
+      relationship_colleague = Neo4j::Http::Relationship.new(label: "KNOWS", primary_key_name: "uuid", uuid: "ColleagueUuid", how: "colleague")
+      edge_b = create_relationship(from, relationship_colleague, to)
+
+      expect(edge_a["relationship"]["uuid"]).to eq("FriendUuid")
+      expect(edge_a["relationship"]["how"]).to eq("friend")
+
+      expect(edge_b["relationship"]["uuid"]).to eq("ColleagueUuid")
+      expect(edge_b["relationship"]["how"]).to eq("colleague")
+
+      result = client.find_relationships(relationship: relationship, from: from, to: to)
```

Comment:
To make sure I understand this, the `relationship` (declared [here](https://github.com/doximity/neo4j-http/pull/7/files#diff-e07944a2cab7b398c9cbef2565f83888e46c96068636a62de2d260b25a532dc8L11)) is only being told the label so both match. But if we wanted only the `{ how: "friend" }` this would be passed into the `relationship` instantiation?